### DISCUSSION TITLE
Is the 7390 with v06.88 secure?

### DISCUSSION THREAD

**Discussion by nicks-technik** [Freetz-NG/freetz-ng#1062] [Allgemein]
Hi, 
i made my first freetz-ng image. It is a 7390_06.88.ger_freetz-ng image. 
I have an old unused AVM Fritzbox 7390. I like to use it in the future for a new slow internet DSL access in Germany. But from the HW the 7390 is more than fast enough. 
But the v06.88 is very old. Are there are still bugfixes in the newly generated freetz images?
Can I use it on the internet with good security?

Thank you for clarification.

**Comment by fda77** [comment]
imho: no. Every Fritzos contains many closed-source components. so there is no public code scanning for security bugs

https://github.com/Freetz-NG/freetz-ng/discussions/424 and https://github.com/Freetz-NG/freetz-ng/discussions/846

To get an official statement, ask AVM!
